### PR TITLE
Follow JS best practices

### PR DESCRIPTION
**Pull request by ab-noori**
## To Do list: interactive list

  ### 🌟Branch features:

  - Webpack configured [`webpack.config.js`](https://github.com/ab-noori/Todo-List-for-Review/blob/9a4c0fce434f7e52d1746338b5d0cca63e57f576/webpack.config.js).
  - HTML structure added [`template.html`](https://github.com/ab-noori/Todo-List-for-Review/blob/9a4c0fce434f7e52d1746338b5d0cca63e57f576/src/template.html) in the source folder.
  - HTML structure styled via SCSS file in the source folder.
  - Task module, UI module, and Store module are added
  - The blueprint of the task object is created by the Task module
  - Add, Remove, and Edit functions are added in its related modules
  - Status module added to check tasks as completed and remove all completed tasks.
  - Filter method is used to filter all completed tasks.
  - > - Data inserts and updates by keypress ENTER

  ### [ 🚀 Branch Demo](https://ab-noori.github.io/Todo-List-for-Review/dist)

### CODE REVIEW

File: src/modules/ui.js [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+      // add book to UI
+      UI.addTaskToList(task);
+
+      // add book to store
+      Store.addTask(task);
+
+      // clear fields
+      UI.clearFields();
+    }
+  }
```
Comment (bronnie313) [inline src/modules/ui.js:28]: while comments can be helpful to document the purpose and functionality of our  code 👍  ,  too much use comments don't make our codebase look good so I kindly request you remove some of the comments in your code

File: src/index.js [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+// Event: update a task
+document.querySelector('#todo-list').addEventListener('keypress', (e) => {
+  // prevent default submit
+
+  if (e.key === 'Enter' && e.target.classList.contains('edit-task')) {
+    // prevent default submit
+    e.preventDefault();
+    // update to the store
+    Store.updateTask(e.target);
+    window.location.reload();
```
Comment (bronnie313) [inline src/index.js:55]: I have noticed that there is a redundancy or repetition of comments in your code, I suggest that we remove some and leave only those that provide additional information to our code that will make our code look good and readable 👍

**Review by bronnie313 — CHANGES_REQUESTED** [review]
Hi Abdul 👋

# Changes Required ♻️ 

I really appreciate the hard work you've put into this code 👍 . I've been going through it, and I've noticed that there are a few points that need you're attending in order to move to the next level 

image

### Highlights 
✔ All linters are checked
✔ Proper PR description 


Please check the comment section for a detailed review thank you 😊😊

File: src/index.js (hunk 2 above)
Comment (ab-noori) [reply src/index.js]: Hi @bronnie313,
Thanks, a lot 🙏  for the time and effort you put to review my project I will make the changes and after that, I will let you know.


thank-you-gif-22

Comment (ab-noori) [reply src/index.js]: image

**Review by bronnie313 — APPROVED** [review]
Hi @ab-noori 👋

# STATUS: APPROVED 🟢

download


**Highlights**

✔ Descriptive PR
✔ Passing Linters
✔ Great implementation of all the requirements requested by the previous reviewer

Amazing work you have done so far 👍  

Your Project is complete there's nothing else to say other than ..it's time to merge it 


Congratulations 💯  
## HAPPY CODING AND ENJOY YOUR DAY 😀😀😀

**Pull request merged**